src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -797,6 +799,8 @@ private static String processPageSourceForGodMode(String content) {
                 .replaceAll(new SimpleDateFormat("EEE, dd MMM yyyy, ").format(now) + "[0-9]{2}:[0-9]{2}", "{*}")
                 // now (used in comments last edited date) e.g. [Thu May 07 07:52:13 UTC 2015]
                 .replaceAll(new SimpleDateFormat("EEE MMM dd ").format(now) + "[0-9]{2}:[0-9]{2}:[0-9]{2} UTC [0-9]{4}", "{*}")
+                // dynamic feedback submission numbers
+                .replaceAll("(?s)<span class=\"submissionsNumber\".*?</span>", "<span class=\"submissionsNumber\" id=\"submissionsNumber\">{*}</span>")
```

Comment:
I am not sure, but do you think by specifying "[0-9]+,[0-9]+" instead of ".*?" the comparison would be faster?